app/projects/config.ts

```diff
@@ -0,0 +1,27 @@
+import reactLogo from '~/images/react-logo.svg'
+import solidLogo from '~/images/solid-logo.svg'
+import vueLogo from '~/images/vue-logo.svg'
+
+export const repo = 'tanstack/config'
+
+export const latestBranch = 'main'
+export const latestVersion = 'v0'
+export const availableVersions = ['v0']
+
+export const colorFrom = 'from-gray-500'
+export const colorTo = 'to-gray-700'
+export const textColor = 'text-gray-700'
+
+export const frameworks = {
+  react: { label: 'React', logo: reactLogo, value: 'react' },
+  solid: { label: 'Solid', logo: solidLogo, value: 'solid' },
+  vue: { label: 'Vue', logo: vueLogo, value: 'vue' },
+} as const
```

Comment:
We don't have any frameworks 😅

Reply:
Yeah, this annoys me too - unfortunately the logic in `<DocsLayout />` currently requires the frameworks object to have items, or it crashes. I was hoping to refactor this in a separate PR.

Since React, Solid and Vue are frameworks we at least recommend using Config's build with, I figured it was a suitable compromise for now. Lmk if you'd prefer we fix this first!

Reply:
Let's try to fix first, otherwise I think it'll lead to a confusing UX for new users trying to figure out what config was

Reply:
Fixed in https://github.com/TanStack/tanstack.com/pull/186